packages/manager/modules/dbaas-logs/src/logs/detail/inputs/add/configure/logs-inputs-add-configure.controller.js

```diff
@@ -19,6 +19,7 @@ export default class LogsInputsAddConfigureCtrl {
     this.$translate = $translate;
     this.serviceName = this.$stateParams.serviceName;
     this.inputId = this.$stateParams.inputId;
+    this.exposedPort = this.$stateParams.exposedPort;
```

Comment:
```suggestion
    this.exposedPort = this.$transition$.params().exposedPort;
```

Reply:
As `$stateParams` is used before I think we could keep it.